.github/workflows/pull_request_gradle.yml

```diff
@@ -28,6 +28,13 @@ jobs:
         java-version: [ 17 ] # Only the first supported LTS; already too many jobs here.
     timeout-minutes: 120
     steps:
+      # Clone timefold-quickstarts
+      # No need to check for stale repo, as Github merges the main repo into the fork automatically.
```

Comment:
Please remove this line (and others like it), we no longer check for stale repo anywhere.